<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jsj_xsmldlg</class>
 <widget class="QWidget" name="jsj_xsmldlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1570</width>
    <height>630</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1570</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDateTimeEdit{
font: 14pt &quot;黑体&quot;;
background:transparent;

border:1px solid #A0A0A0;

border-radius:2px;

min-height:30px;

max-height:30px;
	background-color: rgb(18, 76, 106);
	color: rgb(255, 255, 255);

}
QComboBox{
	background-color: rgb(18, 76, 106);
	color: rgb(255, 255, 255);
font: 14pt &quot;黑体&quot;;
	border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
}
QComboBox::down-arrow {

image:url(:/new/jsj/rc/新增/下拉箭头.png);

 }

 QComboBox::drop-down {

     border:none;

 padding-right:4px;

 }

QComboBox QAbstractItemView{

  outline:0px;

color:rgb(220,220,220);

background-color:rgb(37,86,159);

border:1px solid rgba(143,211,249,127);

border-bottom-right-radius:4px;

border-bottom-left-radius:4px;

 }

QComboBox QAbstractItemView::item{

height:26px;

background-color:rgba(143,211,249,38);

 }
QComboBox QAbstractItemView::item:selected{

background-color:rgb(26,138,219);
 }
QLineEdit{
	
	background-color: rgb(18, 76, 106);
	color: rgb(255, 255, 255);
border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
	border-radius: 3px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */

font: 14pt &quot;黑体&quot;;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-10</y>
     <width>1621</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1621</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1621</width>
     <height>100</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox#groupBox{

	
	border-image: url(:/new/jsj/rc/限速命令/背景一.png);
	border-radius: 2px; /* 边框圆角 */
border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
font: 14pt &quot;黑体&quot;;

}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>1483</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>限速车站</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>命令号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ml_NumberEdit">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="placeholderText">
            <string>请输入5-6位命令号</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>115</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>115</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;黑体&quot;;</string>
         </property>
         <property name="text">
          <string>限速值(Km/h)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Xs_Combo">
         <property name="minimumSize">
          <size>
           <width>145</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>145</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
font: 14pt &quot;黑体&quot;;
	border: 1px solid #A0A0A0; /* 边框宽度为1px，颜色为#A0A0A0 */
}
QComboBox::down-arrow {

image:url(:/new/jsj/rc/新增/下拉箭头.png);

 }

 QComboBox::drop-down {

     border:none;

 padding-right:4px;

 }

QComboBox QAbstractItemView{

  outline:0px;

color:rgb(220,220,220);

background-color:rgb(37,86,159);

border:1px solid rgba(143,211,249,127);

border-bottom-right-radius:4px;

border-bottom-left-radius:4px;

 }

QComboBox QAbstractItemView::item{

height:26px;

background-color:rgba(143,211,249,38);

 }

 QComboBox QAbstractItemView::item:selected{

background-color:rgb(26,138,219);
 }</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;黑体&quot;;</string>
           </property>
           <property name="text">
            <string>限速开始时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="ml_Send_Time">
           <property name="minimumSize">
            <size>
             <width>185</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>185</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>40</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 14pt &quot;黑体&quot;;</string>
             </property>
             <property name="text">
              <string>限速结束时间</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="ml_End_Time">
             <property name="minimumSize">
              <size>
               <width>185</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>185</width>
               <height>32</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QDateTimeEdit{
font: 14pt &quot;黑体&quot;;
background:transparent;

border:1px solid #A0A0A0;

border-radius:2px;

min-height:30px;

max-height:30px;

}
</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Maximum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>1571</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
